test/verify/check-machines

```diff
@@ -370,6 +380,71 @@ class TestMachines(MachineCase):
             print("Libvirt version does not support disk statistics")
             b.wait_present("#vm-{0}-disksstats-unsupported".format(name))
 
+    def testLibvirt(self):
+        b = self.browser
+        m = self.machine
+
+        libvirtServiceName = self.getLibvirtName()
+
+        def checkLibvirtEnabled():
+            try:
+                m.execute("systemctl -q is-enabled {0}".format(libvirtServiceName))
+                return True
+            except:  # exception if return code != 0
```

Comment:
No blanked `except:` please. `except subprocess.CalledProcessError:` sounds appropriate.

Or even better, you can pass the expected state as an argument, and in case of false prepend a `!` to the command. Then you don't need any exception handling.